packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -535,12 +557,13 @@ export class TokenRatesController extends PollingControllerV1<
       return {};
     }
 
-    return Object.entries(tokenPricesByTokenContractAddress).reduce(
-      (obj, [tokenContractAddress, tokenPrice]) => {
+    return Object.entries(contractExchangeRates).reduce(
+      (obj, [tokenContractAddress, tokenValue]) => {
         return {
           ...obj,
-          [tokenContractAddress]:
-            tokenPrice.value * fallbackCurrencyToNativeCurrencyConversionRate,
+          [tokenContractAddress]: tokenValue
+            ? tokenValue * fallbackCurrencyToNativeCurrencyConversionRate
```

Comment:
Realistically, no, but because I've changed `#fetchAndMapExchangeRatesForUnsupportedNativeCurrency` to use `#fetchAndMapExchangeRatesForSupportedNativeCurrency`, `contractExchangeRates` is now a type of ContractExchangeRates, and that is defined as:

``` typescript
interface ContractExchangeRates {
  [address: string]: number | undefined;
}
```

So that means that `tokenValue` here is `number | undefined`.

Reply:
Interesting.... that `| undefined` seems like a mistake, I don't see a case where it is undefined.

Not a blocker though, this doesn't really hurt